src/logging.c

```diff
@@ -517,6 +528,24 @@ void rcutils_logging_console_output_handler(
       token_expansion = name;
     } else if (strcmp("message", token) == 0) {
       token_expansion = message_buffer;
+    } else if (strcmp("seconds", token) == 0) {
+      if (snprintf(
+          numeric_storage, sizeof(numeric_storage),
+          "%f", timestamp / 1e9) == 0)
```

Comment:
Think this should check for `< 0`

Reply:
You're right, but maybe `<= 0`? If it returns zero then it means it wrote no bytes, which also seems like an issue.

Reply:
agreed!

Reply:
I don't think I can test this case, since I get no feedback (other than output to stderr) when this occurs (there's no return code from the function).

Reply:
I fixed the comparisons here: 7f1a17b